src/lib/button-toggle/button-toggle.ts

```diff
@@ -98,10 +98,12 @@ export class MdButtonToggleGroup implements AfterViewInit, ControlValueAccessor
   @ContentChildren(forwardRef(() => MdButtonToggle))
   _buttonToggles: QueryList<MdButtonToggle> = null;
 
+  /** @docs-private */
   ngAfterViewInit() {
```

Comment:
All the lifecycle hooks should be already filtered in [this processor](https://github.com/angular/material2/blob/master/tools/dgeni/processors/docs-private-filter.js#L6-L14)